tools/cdb2sql/cdb2sql.c

```diff
@@ -111,6 +111,11 @@ void dumpstring(FILE *f, char *s, int quotes, int quote_quotes)
         fprintf(f, "'");
 }
 
+static inline void verbose_print(const char *str)
```

Comment:
I'd prefer to keep these changes out of this patch. These are irrelevant and might need more work. For example, the `verbose_print` function does not support variable length arguments which makes it not quite useful.